src/lib/Engineer.test.js

```diff
@@ -0,0 +1,50 @@
+// Engineer Test Cases
+const Engineer = require("../lib/Engineer");
+
+//Mock Data
+describe("Engineer", () => {
+	const mockName = "Barry";
+	const mockId = "12345678";
+	const mockEmail = "test@gmail.com";
+	const mockGithubUsername = "barrylikescoding123";
+	const mockRole = "Engineer";
+
+	//Instantiation of a new instance of the Engineer class
+	const engineer = new Engineer(
```

Comment:
Your engineer class doesn't take `role` as a parameter, so there is no need to pass it in.